build/BuildDefaults.props

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <CLITargets Condition=" '$(CLITargets)' == '' ">Prepare;Compile;Test;Package;Publish</CLITargets>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <HasAdditionalSharedFramework Condition="'$(Rid)' != 'linux-x64'">true</HasAdditionalSharedFramework>
+    <HasAdditionalSharedFramework Condition=" '$(HasAdditionalSharedFramework)' == '' ">false</HasAdditionalSharedFramework>
```

Comment:
It seems like we can just remove this property, and just use `IncludeAdditionalSharedFrameworks`.

I'm not sure the value of this property if you are defaulting it to false.  It is functionally equivalent to `IncludeAdditionalSharedFrameworks`.